.eslintrc.js

```diff
@@ -33,37 +70,63 @@ module.exports = {
       },
     },
     {
-      'files': ['*.test.js', '*.test.jsx'],
+      'files': ['test/**/*.test.js'],
       'rules': {
         'jest/expect-expect': 'off',
         'jest/no-conditional-expect': 'off',
         'jest/no-done-callback': 'off',
         'jest/no-standalone-expect': 'off',
         'jest/no-try-expect': 'off',
-        'no-unused-vars': 'off',
       },
     },
     {
-      'files': ['**/*.ts', '**/*.tsx'],
-      'env': {
-        'node': true,
+      'files': ['test/**/cloudant.v1.test.js'],
+      'rules': {
+        'prettier/prettier': 'off',
+        'object-shorthand': 'off',
       },
-      'extends': ['airbnb-typescript/base', 'prettier'],
+    },
+    // TypeScript specific rules
```

Comment:
Great you have separated js rules from ts!

Reply:
Thanks! 🙂